src/main/DAL/MetaQuestion/MetaQuestionRepository.js

```diff
@@ -1,16 +1,21 @@
 const defineMetaQuestionModel = require("./MetaQuestion")
 const defineAnswerModel = require("./Answer")
 const defineKeywordModel= require("./Keyword")
+const defineAppendixModel = require("./Appendix");
+const {EMSError, MQ_PROCESS_ERROR_CODES: ERROR_CODES} = require("../../EMSError");
+const {MQ_PROCESS_ERROR_MSGS : ERROR_MSGS} = require("../../ErrorMessages");
 
 class MetaQuestionRepository {
     #MetaQuestion
     #Answer
     #Keyword
+    #Appendix
 
     constructor(sequelize) {
         this.#MetaQuestion = defineMetaQuestionModel(sequelize);
         this.#Answer = defineAnswerModel(sequelize);
```

Comment:
we changed the name to key (keys)

Reply:
Isent there #Distructors?

Reply:
what are you talking about? `key` is a type of an answer, not a type of appendix

Reply:
I talked about #Asnwers

Reply:
answered in different conversation